<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectorWidget</class>
 <widget class="QWidget" name="DetectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetectorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QGroupBox" name="vehGroupBox">
     <property name="title">
      <string>参数标志</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="presenceCheckBox">
        <property name="text">
         <string>Det_Presence</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="frontEndsCheckBox">
        <property name="text">
         <string>Det_FrontEnds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="rearEndsCheckBox">
        <property name="text">
         <string>Det_RearEnds</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="frontEndTimeCheckBox">
        <property name="text">
         <string>Det_FrontEndTime</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="rearEndTimeCheckBox">
        <property name="text">
         <string>Det_RearEndTime</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="occupancyRateCheckBox">
        <property name="text">
         <string>Det_OccupancyRate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="occupancyRateSmoothedCheckBox">
        <property name="text">
         <string>Det_OccupancyRateSmoothed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="gapTimeCheckBox">
        <property name="text">
         <string>Det_GapTime</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="vehSpeedCheckBox">
        <property name="text">
         <string>Det_VehSpeed</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="vehLengthCheckBox">
        <property name="text">
         <string>Det_VehLength</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="occupancyTimeCheckBox">
        <property name="text">
         <string>Det_OccupancyTime</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="BusGroupBox">
     <property name="title">
      <string>参数标志</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="telegramsCheckBox">
        <property name="text">
         <string>CP_Telegrams</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="tele_CPCheckBox">
        <property name="text">
         <string>CP_Tele_CP</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="tele_CourseCheckBox">
        <property name="text">
         <string>CP_Tele_Course</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="tele_RouteCheckBox">
        <property name="text">
         <string>CP_Tele_Route</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="tele_LengthCheckBox">
        <property name="text">
         <string>CP_Tele_Length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="tele_ManualDCCheckBox">
        <property name="text">
         <string>CP_Tele_ManualDC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="tele_DelayCheckBox">
        <property name="text">
         <string>CP_Tele_Delay</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="tele_PassengersCheckBox">
        <property name="text">
         <string>CP_Tele_Passengers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="tele_PrioCheckBox">
        <property name="text">
         <string>CP_Tele_Prio</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="tele_LineCheckBox">
        <property name="text">
         <string>CP_Tele_Line</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>名称：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>请求相位号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="phaseComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>请求相位类型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="phaseTypeComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>启用：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="isUsedCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
